cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -878,3 +490,387 @@ def _act_on_(self, args: 'cirq.ActOnArgs', qubits: Sequence['cirq.Qid']) -> bool
             return NotImplemented
 
         return NotImplemented
+
```

Comment:
While you're here, the sig for `_act_on_` should be `_act_on_(self, args: 'cirq.OperationTarget', qubits: Sequence['cirq.Qid']) -> bool`

Reply:
Sure, will update it in this PR as well.